packages/components/src/components/dialog/dialog.tsx

```diff
@@ -235,7 +235,7 @@ export class Dialog extends LitElement {
   /**
    * Specifies the width of the component.
    *
-   * @deprecated Use the `width` property instead.
+   * @deprecated in v3.0, removal target v6.0 - Use the `width` property instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use the `width` property instead.
```